mapStyles2.js

```diff
@@ -0,0 +1,331 @@
+var mapStyles = [
```

Comment:
I assumed this is a google thing, but what is the reason for two separate files? Won't one over write the other?

Reply:
Answered above. Not a google thing, but 2 different files for 2 different styles.